lib/blockscore/util.rb

```diff
@@ -10,6 +10,10 @@ module Util
       'watchlist_hit' => 'watchlist_hits'
     }
 
+    PARSE_ERROR =
+      'An error has occurred. ' \
+        'If this problem persists, please message support@blockscore.com.'
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/59/files#r52251279